_config.yml

```diff
@@ -35,6 +35,9 @@ navigation:
   - title: Sponsors
     url: /sponsors/
     desc: Cha-ching
+  - title: CFP
+    url: /cfp/
+    desc: Be one of our speakers
```

Comment:
is this too many things in the nav now? wondering if we should just link to it from the speakers page